apps/meteor/ee/app/livechat-enterprise/server/api/monitors.ts

```diff
@@ -5,7 +5,10 @@ import { findMonitors, findMonitorByUsername } from './lib/monitors';
 
 API.v1.addRoute(
 	'livechat/monitors',
-	{ authRequired: true },
+	{
+		authRequired: true,
+		permissionsRequired: { GET: { permissions: ['view-livechat-manager', 'manage-livechat-units'], operation: 'hasAny' } },
```

Comment:
Since we have a permission check here, so can we remove it from `findMonitors` to avoid re-checking?